[2022-05-31 20:20:39,928] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: todoist_load_to_pg.run_this_first manual__2022-05-31T20:20:38.197769+00:00 [queued]>
[2022-05-31 20:20:39,942] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: todoist_load_to_pg.run_this_first manual__2022-05-31T20:20:38.197769+00:00 [queued]>
[2022-05-31 20:20:39,943] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 20:20:39,943] {taskinstance.py:1244} INFO - Starting attempt 1 of 1
[2022-05-31 20:20:39,944] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-05-31 20:20:39,961] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): run_this_first> on 2022-05-31 20:20:38.197769+00:00
[2022-05-31 20:20:39,966] {standard_task_runner.py:52} INFO - Started process 127 to run task
[2022-05-31 20:20:39,975] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'todoist_load_to_pg', 'run_this_first', 'manual__2022-05-31T20:20:38.197769+00:00', '--job-id', '379', '--raw', '--subdir', 'DAGS_FOLDER/todoist_load_to_pg.py', '--cfg-path', '/tmp/tmpt4vn2469', '--error-file', '/tmp/tmp8xuwc2_8']
[2022-05-31 20:20:39,976] {standard_task_runner.py:77} INFO - Job 379: Subtask run_this_first
[2022-05-31 20:20:40,044] {logging_mixin.py:109} INFO - Running <TaskInstance: todoist_load_to_pg.run_this_first manual__2022-05-31T20:20:38.197769+00:00 [running]> on host bd658c040114
[2022-05-31 20:20:40,167] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=emil
AIRFLOW_CTX_DAG_ID=todoist_load_to_pg
AIRFLOW_CTX_TASK_ID=run_this_first
AIRFLOW_CTX_EXECUTION_DATE=2022-05-31T20:20:38.197769+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-31T20:20:38.197769+00:00
[2022-05-31 20:20:42,174] {python.py:175} INFO - Done. Returned value was: None
[2022-05-31 20:20:42,191] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=todoist_load_to_pg, task_id=run_this_first, execution_date=20220531T202038, start_date=20220531T202039, end_date=20220531T202042
[2022-05-31 20:20:42,313] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-31 20:20:42,422] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
